include/FadeButton.h

```diff
@@ -35,15 +35,18 @@ class FadeButton : public QAbstractButton
 {
 	Q_OBJECT
 public:
-	FadeButton( const QColor & _normal_color, const QColor &
-					_activated_color, QWidget * _parent );
+	FadeButton( const QColor & _normal_color,
+		const QColor & _activated_color,
+		const QColor & _hold_color,
```

Comment:
In our coding convention:
```
• Function parameters are not prefixed with '_' anymore.
```
It's okay to leave old parameter names as-is, but you shouldn't use underscores in new parameters.
```suggestion
		const QColor & holdColor,
```